Resources/frontend/js/jquery.adyen-payment-selection.js

```diff
@@ -60,7 +54,17 @@
              *
              * @type {String}
              */
-            paymentMethodFormSubmitSelector: 'button[type=submit]',
+            paymentMethodFormSubmitSelector: '#shippingPaymentForm button[type=submit]',
+            /**
+             * @type {string} the group name of Gift card types
+             */
+            giftCardGroupName: 'Gift Card',
+            /**
+             * @type {string} Snippets associated with the payment page
+             */
+            adyenSnippets: {
+                updatePaymentInformation: 'Update your payment information'
```

Comment:
shouldn't this be translated?

Reply:
Yes, the S5 snippets are to be translated by the merchant in the admin panel, that message is the default if the translation is deleted.